cobra/test/test_solver_model.py

```diff
@@ -674,15 +674,15 @@ def test_objective(self, model):
         assert obj.direction == "max"
 
     def test_change_objective(self, model):
-        expression = 1.0 * model.solver.variables['ENO'] + \
-                     1.0 * model.solver.variables['PFK']
-        model.solver.objective = model.solver.interface.Objective(
+        expression = 1.0 * model.variables['ENO'] + \
+                     1.0 * model.variables['PFK']
+        model.solver.objective = model.problem.Objective(
```

Comment:
model.objective integrates with context manager, model.solver.objective does not. For tests we should use model.objective I guess...